### PR TITLE
tools: Properly munge the patternfly font paths

### PR DESCRIPTION
**Pull request by stefwalter**
The OpenSans fonts are in our static directory and used on the
login page. These are not part of a package. In addition we don't
ship any other font formats but woff.

The patternfly.sed script munges patternfly for our use cases,
however it wasn't working with the new patternfly.

Update the script and add tests.

### CODE REVIEW

**Comment by martinpitt** [comment]
This seems to trigger a weird endless loop in semaphore in test-static-code.

File: tools/test-static-code [hunk 1]
```diff
@@ -80,4 +81,16 @@ else
     echo "ok 4 unsafe-security-policy"
 fi
 
+#
+# Bad paths or modifications in patternfly for fonts
+#
+
+if grep -R "\.\./fonts/OpenSans\|fonts/.*eot\|truetype" $distdir/dist/*/*.css >&2; then
+    echo "ERROR: invalid patternfly fonts paths found"
```
Comment (martinpitt) [inline tools/test-static-code]: This caused the endless loop, due to confusing TAP. This message needs to go to `>&2`.

File: tools/test-static-code [hunk 2]
```diff
@@ -80,4 +81,16 @@ else
     echo "ok 4 unsafe-security-policy"
 fi
 
+#
+# Bad paths or modifications in patternfly for fonts
+#
+
+if grep -R "\.\./fonts/OpenSans\|fonts/.*eot\|truetype" $distdir/dist/*/*.css >&2; then
```
Comment (martinpitt) [inline tools/test-static-code]: I wonder  if that is really right. We ship the css in the *source* dir, as part of `make dist`, not in the build dir. So I believe this should just check `dist/*/*.css`? The build directory only has the `*.css.gz` variants.

File: tools/test-static-code [hunk 3]
```diff
@@ -2,8 +2,9 @@
 # run static code checks like pyflakes and pep8
 set -eu
 
-echo "1..4"
+echo "1..5"
 
+distdir=$(pwd)
```
Comment (stefwalter) [inline tools/test-static-code]: This really should say ```builddir=$(pwd)```

File: tools/test-static-code (hunk 2 above)
Comment (stefwalter) [reply tools/test-static-code]: I guess it needs to check both? During development the $builddir/dist/*/*.css exists for me. Can we do both?

**Comment by martinpitt** [comment]
I pushed a FIXUP for the actual changes, and amended your commit to fix the typos and add the Closes #. @stefwalter , please review and check my reasoning?

**Comment by martinpitt** [comment]
Force-pushed my FIXUP again to drop the now obsolete `$distdir` declaration.

File: tools/test-static-code (hunk 2 above)
Comment (martinpitt) [reply tools/test-static-code]: OK, force-pushed my FIXUP again to do both.

**Comment by martinpitt** [comment]
Remaining failures are unrelated.

**Pull request merged**